recipes/pyMCR/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyMCR" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9a2b7c9e1e002d1b2b6f952b1c3cb176e6e9cdb3c685b31431536c817127008
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python  >=3
+    - pytest-runner
+  run:
+    - numpy
+    - python  >=3
+    - scipy
+    - scikit-learn  >=0.20
+
+test:
+  imports:
+    - pymcr
+    - pymcr.tests
+  requires:
+    - pytest
+  commands:
+    - pytest --pyargs pymcr
+
+
+about:
+  home: https://github.com/usnistgov/pyMCR
+  license: NIST
+  license_family: PUBLIC-DOMAIN
+  license_file: LICENSE.md
+  summary: Multivariate Curve Resolution in Python
+  doc_url: https://pages.nist.gov/pyMCR
+  dev_url: https://github.com/usnistgov/pyMCR
+
+extra:
+  recipe-maintainers:
+    - CCampJr
```

Comment:
@CCampJr, would you be happy to be a recipe maintainer for pyMCR?

Reply:
I'd be happy to; though, I'm not 100% sure what that entails

Reply:
I means you have write access to the git repository used for the package (the "feedstock") after you accept the invite which will be automatically sent when this is merged.

Generally the update bot makes pull requests automatically and you just merge them when they pass. You should only need to manually intervene if the dependencies change and if you have issues or questions you can ping @ conda-forge/core (without the space) or ask on [Gitter](https://gitter.im/conda-forge/conda-forge.github.io).

Reply:
@chrisburr Thank you very much!